deployments/kubernetes/charts/clowder2/values.yaml

```diff
@@ -197,19 +197,19 @@ heartbeat:
 ## image        : dokeer/image:tag - the extractor docker image
 ## replicaCount : number - the number of this extractor to install
 ## env          : additional environment variables to pass to extractor.
-extractors:
```

Comment:
in the overrides, you can just say:
```
extractors: []
```
and that should override all extractors listed here, not need to comment them out.